src/django_components/templatetags/component_tags.py

```diff
@@ -217,6 +243,15 @@ def do_component(parser: Parser, token: Token) -> ComponentNode:
     return component_node
 
 
+def do_component_tag_block(parser: Parser, token: Token) -> ComponentNode:
```

Comment:
These two component tag functions intentionally don't have `@register.tag("component")` decorator. Instead, we call these functions at component registration from within `ComponentRegistry`, after we have obtained start and end tags from `TagFormatter`.